apps/interactor/tests/commander/commands/test_effects.py

```diff
@@ -487,6 +486,47 @@ def __init__(s):
             json_output=results.effects_stopped,
         )
 
+    async it "can start linear effects and power on", fake, server, results:
+        for device in fake.devices:
+            device.attrs.power = 0
+
+        await server.assertCommand(
+            "/v1/lifx/command",
+            {"command": "effects/run", "args": {"linear_animation": "MOVE", "linear_options": { "power_on": True }}},
+            json_output=results.success
+        )
+
+        for device in fake.devices:
+            if device.serial in ("d073d5000005", "d073d5000006"):
```

Comment:
```suggestion

```
you're targeting all of them with your command and you set them all to zero to start with, so you can test all devices here

Reply:
actually, ignore that, I'm not awake lol